references/video_classification/train.py

```diff
@@ -98,10 +98,11 @@ def evaluate(model, criterion, data_loader, device):
     return metric_logger.acc1.global_avg
 
 
-def _get_cache_path(filepath):
+def _get_cache_path(filepath, args):
     import hashlib
 
-    h = hashlib.sha1(filepath.encode()).hexdigest()
+    value = f"{filepath}-{args.clip_len}-{args.kinetics_version}-{args.frame_rate}"
```

Comment:
Should we also add `args.step_between_clips` here?

Reply:
We don't have such as parameter on args. If we introduce it, we should absolutely add it here.

Reply:
Got it